packages/hub/services/api-router.ts

```diff
@@ -38,6 +38,7 @@ export default class APIRouter {
   reservationsRoute = inject('reservations-route', { as: 'reservationsRoute' });
   inventoryRoute = inject('inventory-route', { as: 'inventoryRoute' });
   wyrePricesRoute = inject('wyre-prices-route', { as: 'wyrePricesRoute' });
+  checklyWebhookRoute = inject('checkly-webhook-route', { as: 'checklyWebhookRoute' });
```

Comment:
we actually have a route in our server for `application/json` content types specifically to support webhooks (because it is rare that webhooks are actually called with a JSON-API content type). it's the `packages/hub/services/callbacks-router.ts`. I suggest using that router instead.

Reply:
Thanks! Will use that.